src/octopeer-github/content/ElementEventBinding/AuxilaryAddDOMEventFunctions.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Created by Mathias on 2016-06-16.
+ */
+/**
+ * Provide additional functions to help the addDomEvent overrides.
+ */
+class AuxiliaryAddDOMEventFunctions {
+
+    /**
+     * Check whether a rectangle is in scope.
+     * @param width the width of the scope
+     * @param height the height of the scope
+     * @param rectangle the rectangle to be checked
+     * @returns {boolean}
```

Comment:
Can you document when it returns true and when it returns false for all functions?